mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -1226,11 +1229,13 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT {@code super.isInvulnerableTo()} should be called in this case.
 		COMMENT
 		COMMENT @implNote Entity is invulnerable to all damages if it is {@linkplain #isRemoved
-		COMMENT removed}, and is invulnerable to all damages except {@link DamageSource#OUT_OF_WORLD}
+		COMMENT removed}, and is invulnerable to all damages except {@link
+		COMMENT net.minecraft.entity.damage.DamageSources#outOfWorld}
```

Comment:
Same as above, reference the type not the source

Reply:
```suggestion
		COMMENT net.minecraft.entity.damage.DamageTypes#OUT_OF_WORLD}
```